sdk/communication/AzureCommunication/Source/Authentication/CommunicationUserCredential.swift

```diff
@@ -29,13 +29,13 @@ import AzureCore
 #endif
 import Foundation
 
-public typealias AccessTokenRefreshOnCompletion = (AccessToken?, Error?) -> Void
+public typealias AccessTokenRefreshOnCompletion = (CommunicationAccessToken?, Error?) -> Void
 public typealias TokenRefreshOnCompletion = (String?, Error?) -> Void
 
 /**
  The Azure Communication Services User token credential. This class is used to cache/refresh the access token required by Azure Communication Services.
 */
-@objcMembers public class CommunicationUserCredential: NSObject {
+public class CommunicationUserCredential: NSObject {
```

Comment:
Why 'objcMembers' is removed?  i think this is needed for callingSDK to use this class.

Reply:
Yes sorry this was done by mistake